### PR TITLE
Update readhxh.com,readsnk.com,readkingdom.com, etc

### PR DESCRIPTION
**Pull request by wout-codes**
Added few more rules and domains.

### CODE REVIEW

**Review by Alex-302 — COMMENTED** [review]
cannot reproduce. Please provide the links

**Comment by wout-codes** [comment]
@zloyden Would know better for this as he merged similar to this PR some time ago.

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -320,12 +320,14 @@ dosgamers.com##.adsgoogle
...
+readhxh.com,readsnk.com,readkingdom.com,readonepiece.com,readvinlandsaga.com,watchoverlord2.com,demonslayermanga.com,read7deadlysins.com,readfairytail.com,readnaruto.com,watchsao.tv,watchgoblinslayer.com,readdrstone.com,dbsmanga.com,readopm.com,readkaguyasama.com,readjujutsukaisen.com,readtowerofgod.com,readnoblesse.com,readmha.com##.justify-center > div > br:nth-of-type(-n+5)
+readhxh.com,readsnk.com,readkingdom.com,readonepiece.com,readvinlandsaga.com,watchoverlord2.com,demonslayermanga.com,read7deadlysins.com,readfairytail.com,readnaruto.com,watchsao.tv,watchgoblinslayer.com,readdrstone.com,dbsmanga.com,readopm.com,readkaguyasama.com,readjujutsukaisen.com,readtowerofgod.com,readnoblesse.com,readmha.com##.justify-center > div > center
 readopm.com,readkaguyasama.com,readjujutsukaisen.com,readtowerofgod.com,readnoblesse.com##.justify-center > center > a[href] > img
-readopm.com##.widget-area > div.card:first-child
+watchsao.tv,watchgoblinslayer.com,readberserk.com,readopm.com##.widget-area > div.card:first-child
+watchsao.tv,watchgoblinslayer.com,readberserk.com##.pages__ad
+demonslayermanga.com##.text-center > center > a[href] > img[border="0"]
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:330]: If I understand correctly, just this `demonslayermanga.com##a[href^="https://merchoneesan.com/"]` is enough.

Comment (Alex-302) [reply EnglishFilter/sections/specific.txt:330]: without url it may be more universal, if ad will be changed

**Review by zloyden — CHANGES_REQUESTED** [review]
I don't see another social links on `manga.watchsao.tv, manga.watchgoblinslayer.com` except this one (in the screenshot below). They should be unblocked.
<details><summary>Screenshot:</summary>

image

</details><br/>

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/specific.txt:330]: I agree, but as far as I remember, they have had this element for a long time.

**Comment by wout-codes** [comment]
What to do now? Which domains should I unblock the socials?

**Comment by zloyden** [comment]
It's caused by this rule `tokyoghoulre.com,watchgoblinslayer.com,watchnoblesse.com,watchsnk.net,watchtowerofgod.com,watchdrstone.com,demonslayeranime.com,watchblackclover.com,watchkaguyasama.com,haikyuu3.com,watchoverlord2.com,watchboruto.com,watchsao.tv,watchrezero2.com,techsapparel.com,urlty.in,semawur.com,idzone.site##center > a[href] > img`. Just remove `watchsao.tv, watchgoblinslayer.com` from it.

**Comment by wout-codes** [comment]
I don't think it's ideal as it blocks an ad/banner, in some streams. Better make the rule more specific, I suggest.

**Comment by zloyden** [comment]
> I don't think it's ideal as it blocks an ad/banner, in some streams. Better make the rule more specific, I suggest.

This is a different way, but you should provide a link to check the new rule.

**Comment by wout-codes** [comment]
Fixed.

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -4117,7 +4117,7 @@ cloudstorageoptions.com##.adslot_left
 ||adee.hit.gemius.pl/gdejs/xgde.js$domain=sportacentrs.com
 ||webgate.tn/getlink/*.js
 ||webgate.tn/promo/*.gif
-tokyoghoulre.com,watchgoblinslayer.com,watchnoblesse.com,watchsnk.net,watchtowerofgod.com,watchdrstone.com,demonslayeranime.com,watchblackclover.com,watchkaguyasama.com,haikyuu3.com,watchoverlord2.com,watchboruto.com,watchsao.tv,watchrezero2.com,techsapparel.com,urlty.in,semawur.com,idzone.site##center > a[href] > img
+tokyoghoulre.com,watchgoblinslayer.com,watchnoblesse.com,watchsnk.net,watchtowerofgod.com,watchdrstone.com,demonslayeranime.com,watchblackclover.com,watchkaguyasama.com,haikyuu3.com,watchoverlord2.com,watchboruto.com,watchsao.tv,watchrezero2.com,techsapparel.com,urlty.in,semawur.com,idzone.site##center > a[href] > img:not([div.textwidget])
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:4120]: It doesn't work on my end. Try `tokyoghoulre.com,watchgoblinslayer.com,watchnoblesse.com,watchsnk.net,watchtowerofgod.com,watchdrstone.com,demonslayeranime.com,watchblackclover.com,watchkaguyasama.com,haikyuu3.com,watchoverlord2.com,watchboruto.com,watchsao.tv,watchrezero2.com,techsapparel.com,urlty.in,semawur.com,idzone.site##center > a[href] > img:not([class^="alignnone"])`

**Review by zloyden — APPROVED** [review]

**Pull request merged**